net.fsuae.FS-UAE.yml

```diff
@@ -188,8 +188,7 @@ modules:
   - name: appdata
     buildsystem: simple
     build-commands:
-      - mkdir -p ${FLATPAK_DEST}/share/appdata
-      - install -Dm644 net.fsuae.FS-UAE.appdata.xml ${FLATPAK_DEST}/share/appdata/net.fsuae.FS-UAE.appdata.xml
+      - install -D -m644 ${FLATPAK_DEST}/share/metainfo/ ${FLATPAK_ID}.appdata.xml
```

Comment:
either it is `install SOURCE DEST` or `install -t DEST SOURCE` which is also why this is failing.